src/Controller/Api/MediaController.php

```diff
@@ -68,6 +69,33 @@ public function __construct(MediaManagerInterface $mediaManager, Pool $mediaPool
         $this->formFactory = $formFactory;
     }
 
+    /**
+     * Retrieves a specific media.
+     *
+     * @ApiDoc(
+     *  requirements={
+     *      {"name"="id", "dataType"="integer", "requirement"="\d+", "description"="media id"}
+     *  },
+     *  output={"class"="Sonata\MediaBundle\Model\Media", "groups"={"sonata_api_read"}},
+     *  statusCodes={
+     *      200="Returned when successful",
+     *      404="Returned when media is not found"
+     *  }
+     * )
+     *
+     * @Rest\Get("/media/{id}", name="get_medium")
+     *
+     * @Rest\View(serializerGroups={"sonata_api_read"}, serializerEnableMaxDepthChecks=true)
+     *
+     * @param $id
```

Comment:
Missing type.

Reply:
You can add it in UserBundle too. 
https://github.com/sonata-project/SonataUserBundle/pull/1186/files#diff-462edb5816fc8f4ef0c4fafc7c648b3aR132-R136

Reply:
I'm unsure about using `@param int $id`. Regardless the API definition, this is the nearest step previos to declare the type in the argument itself. My concern is related to cases where other type than integers are valid as entity identifiers (like UUIDs).

Reply:
[`User::$id`](https://github.com/FriendsOfSymfony/FOSUserBundle/blob/1049935edd24ec305cc6cfde1875372fa9600446/Model/User.php#L25-L28) is using `mixed`.